instrumentation/opentelemetry-instrumentation-confluent-kafka/tests/test_instrumentation.py

```diff
@@ -284,6 +284,15 @@ def _compare_spans(self, spans, expected_spans):
                     expected_attribute_value, span.attributes[attribute_key]
                 )
 
+    def _assert_topic(self, expected_topic: str) -> None:
+        span_list = self.memory_exporter.get_finished_spans()
+        self.assertEqual(len(span_list), 1)
```

Comment:
Maybe we can separate it into two different functions? assert_span to assert the len of spans or parameterize this function also to get the number of spans to assert?

Reply:
sure, it good idea. how about this?
```python
    def _assert_topic(self, span, expected_topic: str) -> None:
        self.assertEqual(
            span.attributes[SpanAttributes.MESSAGING_DESTINATION],
            expected_topic,
        )

    def _assert_span_count(self, span_list, expected_count: int) -> None:
        self.assertEqual(len(span_list), expected_count)

......
        self.assertIsNotNone(msg)
        span_list = self.memory_exporter.get_finished_spans()
        self._assert_span_count(span_list, 1)
        self._assert_topic(span_list[0], "topic-1")
```